receiver/nsxtreceiver/metadata.yaml

```diff
@@ -0,0 +1,105 @@
+name: nsxtreceiver
+
+resource_attributes:
+  nsxt.node.name:
+    description: The name of the NSX Node.
+    type: string
+  nsxt.node.id:
+    description: The ID of the NSX Node.
+    type: string
+  nsxt.node.type:
+    description: The type of NSX Node.
+    type: string
+    enum:
+      - manager
+      - host
+      - edge
+  nsxt.interface.id:
+    description: The name of the network interface.
```

Comment:
Is this the name of the interface, or the ID?

Reply:
I believe they are somewhat analogous even in NSX-T. I imagine them as both an ID and Name but I can refactor this description to be either/or depending on your preference!

<img width="639" alt="image" src="https://user-images.githubusercontent.com/32067685/167710374-e9c30d42-dcb0-4e82-b0a9-e40bf1096f8d.png">

Reply:
Ah, got it.  Since we are considering using `device.id`, and there isn't an equivalent `device.name`, we can use ID for now.